roles/openshift_metrics/tasks/install_hawkular.yaml

```diff
@@ -1,73 +1,35 @@
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ origin_metrics_dir }}/deployer/templates/{{ item }}.yaml'
+- name: generate replication controller templates
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ mktemp.stdout }}/templates/{{ item }}.yaml"
   with_items:
-  - hawkular-metrics
-  - hawkular-cassandra
-  - hawkular-cassandra-node-pv
-  - hawkular-cassandra-node-dynamic-pv
-  - hawkular-cassandra-node-emptydir
-  - support
+  - hawkular_metrics_rc
+  - hawkular_cassandra_rc
 - name: create hawkular metrics objects
   command: >
     {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ mktemp.stdout }}/templates/{{ file }}'
+    create -f '{{ mktemp.stdout }}/templates/{{ item }}'
   with_items:
   - hawkular_metrics_secrets.yaml
   - hawkular_metrics_certificate.yaml
   - hawkular_metrics_account.yaml
   - cassandra_secrets.yaml
   - cassandra_certificate.yaml
-  loop_control:
-    loop_var: file
-- name: process other templates
-  shell: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    process '{{ file }}'
-    | oc create -f -
-  with_items:
-  - hawkular-cassandra-services
-  - hawkular-support
-  loop_control:
-    loop_var: file
-# TODO persistent storage
-- name: process hawkular-metrics template
-  shell: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    process hawkular-metrics
-    --value 'IMAGE_PREFIX={{ image_prefix }}'
-    --value 'IMAGE_VERSION={{ image_version }}'
-    --value 'METRIC_DURATION={{ metrics_duration }}'
-    --value 'MASTER_URL={{ master_url }}'
-    --value 'USER_WRITE_ACCESS={{ hawkular_user_write_access }}'
-    --value 'USE_PERSISTENT_STORAGE=false'
-    | oc create -f -
-- name: process cassandra template for the master node
+  - hawkular_serviceaccount.yaml
+  - hawkular_metrics_service.yaml
+  - hawkular_metrics_rc.yaml
+  - hawkular_cassandra_service.yaml
+  - hawkular_cassandra_nodes_service.yaml
+  - cassandra_serviceaccount.yaml
+- name: process cassandra template
   shell: >
     {{ openshift.common.client_binary }} -n '{{ project }}'
-    process hawkular-cassandra-node-emptydir
-    --value 'IMAGE_PREFIX={{ image_prefix }}'
-    --value 'IMAGE_VERSION={{ image_version }}'
-    --value NODE=1
-    --value MASTER=true
+    process -f '{{ mktemp.stdout }}/templates/hawkular_cassandra_rc.yaml'
+    --value 'NODE={{ item }}'
+    {% if item == '1' %}--value MASTER=true{% endif %}
     | oc create -f -
-## TODO multi-node cassandra
-#- name: process cassandra template for other nodes
-#  shell: >
-#    {{ openshift.common.client_binary }} -n '{{ project }}'
-#    process hawkular-cassandra-node-emptydir
-#    --value 'IMAGE_PREFIX={{ image_prefix }}'
-#    --value 'IMAGE_VERSION={{ image_version }}'
-#    --value NODE={{ item }}
-#    | oc create -f -
-#  with_sequence: start=1 end={{ cassandra_nodes }}
-#  when: {{ cassandra_nodes > 1 }}
+  with_sequence: start=1 end={{ hawkular_cassandra_nodes }}
```

Comment:
Right, this is a left-over from before when there was a separate step for the master and this was `start=2`.  I will correct it.